src/main/java/jenkins/plugins/slack/workflow/SlackSendStep.java

```diff
@@ -47,6 +47,8 @@
     private static final Logger logger = Logger.getLogger(SlackSendStep.class.getName());
 
     private String message;
+    private boolean uploadFiles;
```

Comment:
why have these been added here? there's a separate step for pipeline `slackUploadFile`